ios/versioned-react-native/ABI34_0_0/EXBarCodeScanner/ABI34_0_0EXBarCodeScanner/ABI34_0_0EXBarCodeScanner.m

```diff
@@ -164,7 +164,7 @@ - (void)captureOutput:(AVCaptureOutput *)captureOutput didOutputMetadataObjects:
     if([metadata isKindOfClass:[AVMetadataMachineReadableCodeObject class]]) {
       AVMetadataMachineReadableCodeObject *codeMetadata = (AVMetadataMachineReadableCodeObject *) metadata;
       for (id barcodeType in _settings[ABI34_0_0EX_BARCODE_TYPES_KEY]) {
-        if ([metadata.type isEqualToString:barcodeType]) {
+        if (codeMetadata.stringValue && [metadata.type isEqualToString:barcodeType]) {
```

Comment:
```suggestion
        if (codeMetadata.stringValue && [codeMetadata.type isEqualToString:barcodeType]) {
```
or 
```suggestion
        if (metadata.stringValue && [metadata.type isEqualToString:barcodeType]) {
```

It's not consistent with what's being changed in `unversioned` package

Reply:
I know